Assets/Scripts/EnemyController.cs

```diff
@@ -16,130 +18,120 @@ public class EnemyController : MonoBehaviour
     public int maxSpeed;
     public int shieldRegen;
     public float weight;
-    public string movementType;
+    public MovementType movementType;
```

Comment:
There's also the benefit of an enum becoming a dropdown in the Unity editor! Go see for yourself; now there's a dropdown in the enemy object in the editor that automatically is populated with "PATH" and "MONGO". This is another "prevent yourself from shooting yourself in the foot" thing that an enum brings to the table, as now you can't typo the editor either.